gamemodes/terrortown/gamemode/shared/sh_init.lua

```diff
@@ -10,6 +10,9 @@ GM.Customized = true
 
 TTT2 = true -- identifier for TTT2. Just use "if TTT2 then ... end"
 
+-- extend CLIENT and SERVER flags
+SHARED = 2
```

Comment:
Is this really needed? You could simply use no argument or `CLIENT or SERVER`. An existing convar like this (handled in another way than CLIENT and SERVER are done) could result in big confusion

Reply:
I thought it was nicer in the code to be honest. But no, it is not really needed.

Reply:
Yea but doing `if SHARED then` will return true EVERYTIME, even if just done on the SERVER or CLIENT.

Reply:
yes, but it wasn't intended for that usage. See my linked example. I somehow have to pass to the function that it is shared. This seemed to be the best solution to me.

Reply:
Then make it a boolean and just use hardcoded `true` or `CLIENT or SERVER` (true)

Reply:
But I need to differentiate between three states, not just two:
- include on the server
- include on the server and distribute to the client
- only distribute to the client

Reply:
local var or no var (nil == shared)

Reply:
To be honest, this is pretty janky.